### PR TITLE
Add tests migrations

### PR DESCRIPTION
**Pull request by Voloodya**
#20 Add tests migrations

### CODE REVIEW

File: src/test/java/org/testcontainers/containers/TarantoolCartridgeMigrationsTest.java [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+                    "Dockerfile",
+                    "cartridge/instances.yml",
+                    "cartridge/topology.lua")
+                    .withDirectoryBinding("cartridge")
+                    .withStartupTimeout(Duration.ofSeconds(300))
+                    .withLogConsumer(new Slf4jLogConsumer(
+                            LoggerFactory.getLogger(TarantoolCartridgeStaticContainerTest.class)));
+
+    @BeforeAll
+    public static void start_container(){
```
Comment (vrogach2020) [inline src/test/java/org/testcontainers/containers/TarantoolCartridgeMigrationsTest.java:41]: Please follow java naming style and test guidelines
https://github.com/tarantool/cartridge-java/blob/master/docs/test-convention.md

File: src/test/java/org/testcontainers/containers/TarantoolCartridgeMigrationsTest.java [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+        }
+
+    }
+
+    @Test
+    public void test_migrations_migrator() throws Exception {
+
+        if(container.isRunning()){
+            try {
+                List<?> resultMigration = container.executeCommand("require('migrator').up()").get();
```
Comment (vrogach2020) [inline src/test/java/org/testcontainers/containers/TarantoolCartridgeMigrationsTest.java:58]: this should be a part of test setup

File: src/test/java/org/testcontainers/containers/TarantoolCartridgeMigrationsTest.java [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+            } catch (Exception e) {
+                e.printStackTrace();
+                throw new RuntimeException(e);
+            }
+        }
+    }
+
+    @Test
+    public void test_migrations_curl(){
```
Comment (vrogach2020) [inline src/test/java/org/testcontainers/containers/TarantoolCartridgeMigrationsTest.java:67]: Do not use curl please to reduce dependencies on external modules

File: src/test/resources/cartridge/migrations/001_spase_test.lua [hunk 4]
```diff
@@ -0,0 +1,32 @@
+return{
+
+    up = function()
+        local utils = require('migrator.utils')
+
+        local routerSessionContexts = box.schema.space.create('spaceTest', { if_not_exists = true })
```
Comment (vrogach2020) [inline src/test/resources/cartridge/migrations/001_spase_test.lua:6]: Don't this complex and meaningless name 'routerSessionContexts' ? use 'test' or something like that

File: src/test/java/org/testcontainers/containers/TarantoolCartridgeMigrationsTest.java [hunk 5]
```diff
@@ -0,0 +1,145 @@
...
+                    .withLogConsumer(new Slf4jLogConsumer(
+                            LoggerFactory.getLogger(TarantoolCartridgeStaticContainerTest.class)));
+
+    @BeforeAll
+    public static void start_container(){
+        if (!container.isRunning()) {
+            try {
+                container.start();
+            } catch (Exception e) {
+                e.printStackTrace();
```
Comment (vrogach2020) [inline src/test/java/org/testcontainers/containers/TarantoolCartridgeMigrationsTest.java:46]: Remove prints please. 
See review checklist and follow code guidelines
https://github.com/tarantool/cartridge-java/blob/master/docs/review-checklist.md

File: src/test/java/org/testcontainers/containers/TarantoolCartridgeMigrationsTest.java [hunk 6]
```diff
@@ -0,0 +1,145 @@
...
+
+        if(container.isRunning()){
+            URL url = null;
+            HttpURLConnection connection = null;
+            OutputStream os = null;
+            InputStreamReader inputStreamReader = null;
+            BufferedReader bfR = null;
+            StringBuilder strBuilder = new StringBuilder();
+
+            Map<String, String> bodyHttpPostRequest = new HashMap<>();
```
Comment (vrogach2020) [inline src/test/java/org/testcontainers/containers/TarantoolCartridgeMigrationsTest.java:94]: Refactor please. Extract all http stuff to a method

File: src/test/resources/cartridge/migrations/001_spase_test.lua [hunk 7]
```diff
@@ -0,0 +1,32 @@
+return{
```
Comment (vrogach2020) [inline src/test/resources/cartridge/migrations/001_spase_test.lua:1]: Fix typo in filename 'spase'

**Review by vrogach2020 — CHANGES_REQUESTED** [review]

File: src/test/java/org/testcontainers/containers/TarantoolCartridgeMigrationsTest.java (hunk 3 above)
Comment (Voloodya) [reply src/test/java/org/testcontainers/containers/TarantoolCartridgeMigrationsTest.java:67]: I don't understand.  Only standart container tools are user here

File: src/test/java/org/testcontainers/containers/TarantoolCartridgeMigrationsTest.java (hunk 2 above)
Comment (Voloodya) [reply src/test/java/org/testcontainers/containers/TarantoolCartridgeMigrationsTest.java:58]: Each migration method is checked separatly, this one too

**Review by vrogach2020 — APPROVED** [review]

**Pull request merged**